test/__fixtures__/substrate-node-api-fn.js

```diff
@@ -22,59 +22,29 @@ module.exports = {
       return true
     },
     get runtimeChain() {
-      return "Test"
+      return 'Test'
     },
     get runtimeVersion() {
       return {
-        specName: "dscp-node",
-        implName: "dscp-node",
+        specName: 'dscp-node',
+        implName: 'dscp-node',
         authoringVersion: 1,
         specVersion: 300,
         implVersion: 1,
         apis: [
-          [
```

Comment:
formatting churn comes from removing `"ignorePatterns": ["**/test/__fixtures__/**/*"],` from `.eslintrc`. Can't see a reason why we were ignoring this directory